### PR TITLE
Awesome Book

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
This is a simple project to add and remove list items using the following;

- Plain Javascript.
- Objects and Arrays.
- Dynamic DOM manipulation and 
- Basic add event methods.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+};
+const addBtn = document.querySelector('.add');
+addBtn.addEventListener('click', (event) => {
+  event.preventDefault();
+  if (title.value !== '' && author.value !== '') {
+    const book = {
+      title,
+      author,
+    };
+    books.push(book);
```
Comment (EricMbouwe) [inline index.js]: - [ ] Let's create a function that adds a new book to the collection, with title and author. It makes your code more readable and improves the separation of tasks.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      div.innerHTML = `
+            <p>${book.title.value}</p>
+            <p>${book.author.value}</p>                                          
+            <button type="button" id="${index}" class="remove">Remove</button>   
+            <hr>`;
+      const removebtn = div.querySelector('.remove');
+      removebtn.addEventListener('click', () => {
+        books.splice(index, 1);
+        div.remove();
+        storage();
```
Comment (EricMbouwe) [inline index.js]: - [ ] Let's create a function that removes a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method. It makes your code more readable and improves the separation of tasks.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎉
- Linters ✔
- Descriptive PR ✔
- Use of DOM selectors ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥✅✅ 


## Highlights ✨
- No Linters error ✅✅
- Good commit messages ✅✅
- Great working on the functionality ✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**